Gems/EMotionFX/Code/Include/Integration/AnimGraphComponentBus.h

```diff
@@ -191,6 +191,9 @@ namespace EMotionFX
             /// Making a request to desync from the anim graph to its leader graph
             /// \param leaderEntityId - the entity id of another anim graph.
             virtual void DesyncAnimGraph(AZ::EntityId leaderEntityId) = 0;
+
+            /// Set the name of the active motion set.
+            virtual void SetActiveMotionSet(const char* activeMotionsetName) = 0;
```

Comment:
nit: `MotionsetName` -> `MotionSetName` (also should the function be called `SetActiveMotionSetName`?

Reply:
@amzn-rhhong did you see my comment about the name of the function? As this isn't actually changing the active motion set but the name of the action motion set, should this instead be called `SetActionMotionSetName`? If you think it's unnecessary it doesn't have to change but it may be clearer at the callsite if called this (so it doesn't seem like we're using the string as a key to lookup a new motion set and set it as the active one).

Reply:
This will change the active motion set, not change the name of the activated motion set. I think the function name is fine, but I understand it's a little confusing.

Reply:
Ah okay understood, thanks for clarifying! 👍